From c088d9752a16fcd1bd7789b7e49cf00d1fd4ec72 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Tue, 16 Oct 2018 12:07:00 +0900
Subject: [PATCH 522/909] arm64: dts: r8a77990-ebisu: Set max dot clock for vga

commit 27aafebfab0452e2025ad480e6cbc206473c1c78 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add option to specify the maximum dot clock that can be
output on VGA connector in Ebisu board.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 10ddfb1ce8dc..f108a6d2eac4 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -172,6 +172,7 @@
 
 		width = <1024>;
 		height = <768>;
+		max-pixelclock = <75000>;
 
 		port {
 			vga_in: endpoint {
-- 
2.17.1

